src/util/endianness_map.h

```diff
@@ -41,6 +41,8 @@ class endianness_mapt
   {
   }
 
+  virtual ~endianness_mapt() = default;
```

Comment:
I agree with @kroening that some classes should rather be made final, such as this one.